ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master__4.9.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  ocp_builder_golang-1.13:
```

Comment:
Latest operator is built using golang:1.16.  Will this need to be changed ?

Reply:
I changed to 1.16